manifests/sysctl.pp

```diff
@@ -149,6 +152,9 @@
   Optional[Boolean]    $ipv6                                           = undef
 ) {
 
+  # include module to set boot time kernel parameters
+  include 'simp::kernel_param'
```

Comment:
I think this would be more suited to another class, like init or scenario::base. Also, can you wrap this in a boolean?